modules/create-policy-from-system-policies-view.adoc

```diff
@@ -8,58 +8,67 @@
+. In the *Policy behavior* section, perform the following steps:
+.. Select the *Lifecycle stages* to which your policy is applicable: *Build*, *Deploy*, or *Runtime*.
 You can select more than one stage.
 *** Build-time policies apply to image fields such as CVEs and Dockerfile instructions.
 *** Deploy-time policies can include all build-time policy criteria but they can also include data from your cluster configurations, such as running in privileged mode or mounting the Docker socket.
 *** Runtime policies can include all build-time and deploy-time policy criteria but they can also include data about process executions during runtime.
-. For *Response method*, either select:
-.. *Inform* to include the violation in the violations list.
-.. Or select *Inform and enforce* to enforce actions.
-*** Choose the enforcement behavior for the policy.
-It is only available for the stages you select when configuring *Lifecycle Stages*.
-Select *ON* (enable) to enforce policy and report a violation, and *OFF* (disable) to only report a violation.
+.. Optional: If you selected the *Runtime* lifecycle stage, select one of the following *Event sources*:
+*** Deployment: {product-title-short} triggers policy violations when event sources include process and network activity, pod exec and pod port forwarding.
+*** {product-title-short} triggers policy violations when event sources match Kubernetes audit log records.
+. For *Response method*, select one of the following options:
+.. *Inform*: include the violation in the violations list.
+.. *Inform and enforce*: enforce actions.
+. Optional: If you selected *Inform and enforce*, in *Configure enforcement behavior*, select the enforcement behavior for the policy by using the toggle for each lifecycle.
+It is only available for the stages you select when configuring *Lifecycle stages*.
 The enforcement behavior is different for each lifecycle stage.
-**** *Build* - {product-title} fails your continuous integration (CI) builds when images match the conditions of the policy.
-**** *Deploy* - {product-title} blocks creation of deployments that match the conditions of the policy.
+**** *Build* - {product-title-short} fails your continuous integration (CI) builds when images match the criteria of the policy.
+**** *Deploy* - {product-title-short} blocks creation of deployments that match the conditions of the policy.
 In clusters with admission controller enforcement, the Kubernetes or {ocp} API server blocks all noncompliant deployments.
-In other clusters, {product-title} edits noncompliant deployments to prevent pods from being scheduled.
-**** *Runtime* - {product-title} kills all pods that match the conditions of the policy or blocks the action taken on the pods.
+In other clusters, {product-title-short} edits noncompliant deployments to prevent pods from being scheduled.
+**** *Runtime* - {product-title-short} kills all pods when an event in the pods matches the criteria of the policy.
```

Comment:
I realize that you didn't add the word "kills" here, but is it possible to replace it with something less violent? Maybe "stops" or "ends"?

Reply:
I've asked the developer to make sure that another word would be accurate since I'm not sure if this is a specific term that's used in Kubernetes and has a specific meaning.